simple-weather/simple-weather/Networking.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Networking.swift
+//  simple-weather
+//
+//  Created by egmars.janis.timma on 24/04/2019.
+//  Copyright © 2019 egmars.janis.timma. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+struct APIService {
+    
+    func getData(coordinates: CLLocationCoordinate2D, completionHandler: @escaping (CurrentWeather) -> Void) {
```

Comment:
Rewrite this using **Swift 5**, use new and fancy [Result](https://www.hackingwithswift.com/articles/161/how-to-use-result-in-swift) type